### DISCUSSION TITLE
Apply to Sovereign Tech Fund?

### DISCUSSION THREAD

**Discussion by LoneRifle** [node-saml/node-saml#375] [General]
I just learned about the [Sovereign Tech Fund](https://www.sovereigntechfund.de/) from the Germany Government, which provides funding for open-source projects considered important to the functioning of the modern Internet.

Given how important node-saml and the other packages in this org are considered fundamental to how Node applications interact with SAML providers, should the maintainers consider making an application to obtain funding, to ensure that stewardship of these packages is better guaranteed?

**Comment by markstos** [comment]
@cjbarth Sounds like a good idea to me.

**Comment by cjbarth** [comment]
Given that the program has a minimum cost of €50,000 for a submitted project to be funded, I don't think we'd qualify as we don't have any work lined up that would be anywhere near that.

https://www.sovereign.tech/programs/fund#requirements
